android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -151,17 +152,27 @@ public void onMethodCall(@NonNull MethodCall call, @NonNull Result result) {
                 break;
             case "purchaseProduct":
                 String productIdentifier = call.argument("productIdentifier");
-                String oldSKU = call.argument("oldSKU");
-                Integer prorationMode = call.argument("prorationMode");
+                String oldSKU = call.argument("oldSKU"); // TODO: Rename
+                Integer prorationMode = call.argument("prorationMode"); // TODO: Rename
+                Boolean isPersonalizedPrice = call.argument("isPersonalizedPrice"); // TODO: Rename
```

Comment:
Going to handle these renames in a followup PR since they involve some kind of deprecations / doc changes